src/core/server/index.ts

```diff
@@ -47,6 +47,7 @@ import { IUiSettingsClient, UiSettingsServiceSetup } from './ui_settings';
 import { SavedObjectsClientContract } from './saved_objects/types';
 import { SavedObjectsServiceSetup, SavedObjectsServiceStart } from './saved_objects';
 import { CapabilitiesSetup, CapabilitiesStart } from './capabilities';
+import { PulseServiceSetup, PulseServiceStart } from './pulse';
```

Comment:
`src/core/server/pulse/index.ts` doesn't have exported interfaces `PulseServiceSetup` and `PulseServiceStart`